### PR TITLE
Introduce InstanceUUID for VirtualMachine

### PR DESCRIPTION
**Pull request by janitha**
Introduce a InstanceUUID field to the VirtualMachine Status which reflect the vSphere Instance UUID.

### CODE REVIEW

File: api/v1alpha1/virtualmachine_types.go [hunk 1]
```diff
@@ -310,6 +310,9 @@ type VirtualMachineStatus struct {
 	// Guest OS BIOS as a unique hardware identifier.
 	BiosUUID string `json:"biosUUID"`
 
+	// InstanceUUID describes the instance UUID provideed by the underlying vSphere infrastructure.
```
Comment (aruneshpa) [inline api/v1alpha1/virtualmachine_types.go:313]: `provideed` -> `provided`.

Comment (aruneshpa) [reply api/v1alpha1/virtualmachine_types.go:313]: Also, should this be `omitempty`?

Comment (sreyasn) [inline api/v1alpha1/virtualmachine_types.go]: LGTM. 'provided' typo

Comment (janitha) [reply api/v1alpha1/virtualmachine_types.go]: Every VM must have an InstanceUUID (similar to BiosUUID and UniqueID), so will leave the omitempty out.

**Review by sreyasn — APPROVED** [review]

File: api/v1alpha1/virtualmachine_types.go (hunk 1 above)
Comment (janitha) [reply api/v1alpha1/virtualmachine_types.go]: Fixed

**Review by aruneshpa — APPROVED** [review]

**Pull request merged**